test/hotel_system_test.rb

```diff
@@ -0,0 +1,211 @@
+require_relative "test_helper"
+
+describe "Hotel_System class" do
+  describe "initialize" do
+    before do
+      @hotel_system = Hotel::HotelSystem.new
+      @rooms = (1..20).to_a
+      @reservations = []
+    end
+
+    it "is an instance of HotelSystem" do
+      expect(@hotel_system).must_be_instance_of Hotel::HotelSystem
+    end
+
+    it "returns an array of rooms" do
```

Comment:
This doesn't actually test returning rooms, it just uses the list from your `before`.